content/blog/tidymodels-2021-q2/index.Rmd

```diff
@@ -0,0 +1,232 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2021-q2
+title: Q2 tidymodels digest
+date: 2021-07-02
+author: Julia Silge
+description: >
+    Releases of tidymodels packages in Q2 of 2021 include more streamlined
+    memory footprints for feature engineering recipes, new model engine options,
+    and better support for post-processing predictions.
+
+photo:
+  url: https://unsplash.com/photos/UsZNiW6LyQ0
+  author: SHOT
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [roundup] 
+tags: [tidymodels, parsnip, recipes, butcher, yardstick, probably]
+---
+
+```{r include=FALSE}
+library(ggplot2)
+theme_set(theme_minimal())
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. Earlier this year, we [started regular updates](https://www.tidyverse.org/blog/2021/03/tidymodels-2021-q1/) here on the tidyverse blog summarizing recent developments in the tidymodels ecosystem overall. You can check out the [`tidymodels`](https://www.tidyverse.org/tags/tidymodels/) tag to find all tidymodels blog posts here, including those that focus on a single package or more major releases, but the purpose of these roundup posts is to keep you informed about any releases you may have missed and useful new functionality as we maintain these packages.
+
+Since our last roundup post, there have been 19 CRAN releases of 15 different packages. That might sound like a lot of change to absorb as a tidymodels user! However, we purposefully write code in small, modular packages that we can release frequently, to make models easier to deploy and our software easier to maintain. You can install these updates from CRAN with:
```

Comment:
```suggestion
Since our last roundup post, there have been 19 CRAN releases of 15 different packages. That might sound like a lot of change to absorb as a tidymodels user! However, we purposefully write code in small, modular packages that we can release frequently to make models easier to deploy and our software easier to maintain. You can install these updates from CRAN with:
```

Reply:
Extraneous comma (aside: I add these little notes when I'm not sure how obvious the change will be in the GitHub UI—let me know if they're simply annoying, and I'll stop)